SmartSaver.MVC/Views/Home/Index.cshtml

```diff
@@ -2,7 +2,40 @@
     ViewData["Title"] = "Home Page";
 }
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
+<html>
+@{var myStyleSheet = "~/css/generalDesign.css";}
+
+<!-- This creates a link to the CSS file. -->
+<link rel="stylesheet" type="text/css" href="@Href(myStyleSheet)" />
+
+<body style="background-color:#f5f5ef">
+    <div class="container-fluid.max-width: 100%;">
+        <div class="row">
+            <div class="col-sm" style="padding:90px 30px 30px 30px; text-align:center; font-family:'Courier New'">
```

Comment:
It would be easier to understand, if instead of inline styles, you would put styling in css file.